.vscode/cspell.json

```diff
@@ -1309,6 +1309,13 @@
         "NFVIs",
         "nfvis"
       ]
+    },
+    {
+      "filename": "**/sdk/confluent/**/*.cs",
+      "words": [
+        "Acces",
```

Comment:
This is not an actual word; please fix the code rather than adding misspellings to the global file.